rs-client-demo/src/main.rs

```diff
@@ -15,6 +16,14 @@ impl Verification for CustomVerifier {
     }
 }
 
+fn print_records(records: Vec<serde_json::Value>) {
```

Comment:
Just FYI -- this consumes `records`!  You want to take `records: &[serde_json::Value]`.

Reply:
yes